scripts/check_binary_dependencies_ebpfapi_dll_regular_debug.txt

```diff
@@ -27,7 +27,13 @@ api-ms-win-core-processthreads-l1-1-1.dll
 api-ms-win-core-profile-l1-1-0.dll
 api-ms-win-core-interlocked-l1-1-0.dll
 api-ms-win-core-heap-l1-1-0.dll
+api-ms-win-core-localization-l1-2-0.dll
```

Comment:
How many and what DLL's does this change pull in?

Reply:
All the new API sets are implemented in kernel32.dll, which was already pulled in.
So nothing new here besides the api set redirects.

Reply:
The dependency changes are now reverted with the new design.